packages/swarm-docs/src/components/nav.js

```diff
@@ -3,21 +3,45 @@ import * as React from 'react';
 import { Link } from 'gatsby';
 
 type Props = {
-    fileNames: Array<string>
+    files: Array<Object>
```

Comment:
Is there any more specific type than `Object` for a `file`?

Reply:
yes! meant to update this and forgot